docker/metricq-clustered/docker-compose.yml

```diff
@@ -101,53 +101,50 @@ services:
     command: "metricq-source-dummy -s amqp://admin:admin@rabbitmq-node0"
 
   rabbitmq-server-node0:
-    image: rabbitmq:management-alpine
+    build:
+      context: .
     environment:
-      RABBITMQ_DEFAULT_USER: "admin"
-      RABBITMQ_DEFAULT_PASS: "admin"
-      RABBITMQ_DEFAULT_VHOST: "/"
-      RABBITMQ_ERLANG_COOKIE: "Leibniz Butterkeks"
       RABBITMQ_NODENAME: "rabbit@rabbitmq-node0"
     networks:
       metricq-network:
         aliases:
           - rabbitmq-server
           - rabbitmq-node0
     ports:
-      - "127.0.0.1:5672:5672"
```

Comment:
I'd rather not expose the cluster on the global network interface of the host, but the loopback interface.